app/controllers/hyrax/batch_edits_controller.rb

```diff
@@ -60,11 +60,30 @@ def update_document(obj)
       obj.save
     end
 
+    def valkyrie_update_document(obj)
+      form = form_class.new(obj, current_ability, nil)
+      return unless form.validate(params[form_class.model_class.model_name.param_key])
+
+      cleanup_form_fields form
+
+      result = transactions['change_set.update_work']
+               .with_step_args('work_resource.save_acl' => { permissions_params: form.input_params["permissions"] })
+               .call(form)
+      obj = result.value!
+
+      InheritPermissionsJob.perform_now(obj)
```

Comment:
Question: Should this be in the transaction?

Reply:
we don't seem to enqueue these two jobs in a normal `update_work` code path. (should we?)

if we need to, i think it wouldn't be in the Transaction, but in an event listener. i.e. should we be triggering these changes whenever `object.acl.updated` gets called? or, if not, why are they needed here?

Reply:
looking closer, i suspect the answer "because that's what the AF implementation does" should be good enough for us here.